docs/source/registrationOverview.rst

```diff
@@ -243,3 +243,10 @@ This is done using the Command-Observer pattern, associating callbacks with spec
 To associate a callback with a specific `event <https://simpleitk.org/doxygen/latest/html/namespaceitk_1_1simple.html#aa7399868984d99493c5a307cce373ace>`_
 use the `AddCommand <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1ProcessObject.html#a2199e5cca19b45d504676a595e1f6cfd>`_
 method.
+
+Reproducibility
+===============
+
+Generally speaking, repeated registrations of the same datasets will yield slightly different results. This is associated with the registration framework's
+similarity metric computation implementation which includes randomization and is multi-threaded. To eliminate the variability due to the internal use of
+randomization, set the seed parameter for the `SetMetricSamplingPercentage <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1ImageRegistrationMethod.html#a8b891c62404a8dc5010241fea619c932>`_ to a fixed value (e.g. `42 <https://en.wikipedia.org/wiki/42_(number)>`_). Eliminating the variability associated with multithreading is a slightly more complicated issue. For full reproducibility we need to set the number of threads to one via the `SetGlobalDefaultNumberOfThreads <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1ProcessObject.html#a305b43330f9feb26325eadfc30695bd9>`_ method. Unfortunatly, this will cause the registration to take significantly longer. From a practical standpoint, in the tradeoff between full reproducibilty and computational efficiency with minor result differences, most of the time computational efficiency is more important.
```

Comment:
^Unfortunatly^Unfortunately^
^reproducibilty^reproducibility^

Reply:
Fixed spelling and refined the presentation to emphasize that most of the variability is due to the use of randomization and only somewhat to the use of multithreading.